rust/sdk-processor/src/config/processor_config.rs

```diff
@@ -120,6 +125,9 @@ impl ProcessorConfig {
                 CurrentTableItem::TABLE_NAME.to_string(),
                 TableMetadata::TABLE_NAME.to_string(),
             ]),
+            ProcessorName::ParquetFungibleAssetProcessor => {
+                HashSet::from([FungibleAssetActivity::TABLE_NAME.to_string()])
```

Comment:
the rest of the fa tables are missing. is this intended?

Reply:
ahh i updated this in the stacked pr, will update this here.